dbgring/dbgring.ml

```diff
@@ -36,33 +36,33 @@ let open_ring domid mfn =
 
 let hexify s =
```

Comment:
There is a package called `hex` in xs-opam that might already do this. Probably not worth adding a dependency right now, but if we see more of these functions in the future we should probably reduce code duplication and replace them with calls to Hex.